app/templates/client/styles/_colors.scss

```diff
@@ -1,6 +1,3 @@
-@import '~react-toolbox/lib/colors';
-
-// Reference react-toolbox colors here https://github.com/react-toolbox/react-toolbox/blob/dev/components/_colors.scss.
-// Define colors with HEX values here - example:
-
 $zeal-orange: #d85226;
+$header-background-color: #424242;
+$header-color: #fff;
```

Comment:
These feel like maybe they should be localized to just the header.

Reply:
We can probably just give them color names rather than variables that were for the headers only. `$white` and `$charcoal`

Reply:
I'd either move them down to where the header is or leave them as-is.